doc/classes/Projection.xml

```diff
@@ -1,27 +1,34 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="Projection" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
+		3D projection (4x4 matrix).
 	</brief_description>
 	<description>
+		A 4x4 matrix used for 3D projective transformations. It can represent transformations such as translation, rotation, scaling, shearing, and perspective division. It consists of four [Vector4] columns.
+		For purely linear transformations (translation, rotation, and scale), it is recommended to use [Transform3D], as it is more performant and has a lower memory footprint.
+		Used internally as [Camera3D]'s projection matrix.
 	</description>
 	<tutorials>
 	</tutorials>
 	<constructors>
 		<constructor name="Projection">
 			<return type="Projection" />
 			<description>
+				Constructs a default-initialized [Projection] set to [constant IDENTITY].
 			</description>
 		</constructor>
 		<constructor name="Projection">
 			<return type="Projection" />
 			<param index="0" name="from" type="Projection" />
 			<description>
+				Constructs a [Projection] as a copy of the given [Projection].
 			</description>
 		</constructor>
 		<constructor name="Projection">
 			<return type="Projection" />
 			<param index="0" name="from" type="Transform3D" />
 			<description>
+				Constructs a Projection with the first 3 rows initialized as a copy of the given [Transform3D].
```

Comment:
This isn't quite accurate. The first three rows and first three columns are the Transform3D's basis, but the last row is the Transform's origin see here:
https://github.com/godotengine/godot/blob/39534a7aecc4ca4215af67244b23dda09ea339f8/core/math/projection.cpp#L914-L934